src/Orleans.Membership.ZooKeeper/ZooKeeperHostingExtensions.cs

```diff
@@ -25,21 +22,15 @@ public static class ZooKeeperHostingExtensions
         /// <summary>
         /// Configure siloHostBuilder with ZooKeeperMembership
```

Comment:
"Configures the silo to use ZooKeeper for cluster membership" - but it's perhaps out of scope for this pr to change the comment